packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -22,6 +22,16 @@ type MetadataParams<TPublic = OrganizationPublicMetadata, TPrivate = Organizatio
 type GetOrganizationListParams = ClerkPaginationRequest<{
   includeMembersCount?: boolean;
   query?: string;
+  orderBy?:
+    | 'name'
+    | '+name'
+    | '-name'
+    | 'created_at'
+    | '+created_at'
+    | '-created_at'
+    | 'members_count'
+    | '+members_count'
+    | '-members_count';
```

Comment:
The available values corresponds to the [ones documented in the Backend API Reference](https://clerk.com/docs/reference/backend-api/tag/Organizations#operation/ListOrganizations!in=query&path=order_by&t=request).